File: tests/fixtures/python/simple_class.py
from typing import Final


class User:
    MAX_LOGIN_ATTEMPTS: Final[int] = 5

    def __init__(self, email: str) -> None:
        self.email: str = email
        self._token: str = ""
        self.__secret: str = "shh"

    @staticmethod
    def normalize_email(email: str) -> str:
        return email.lower()

    def login(self, password: str) -> bool:
        return True

    def __eq__(self, other: object) -> bool:
        return False

File: tests/fixtures/java/simple_class.java
public class User {
    public static final int MAX_LOGIN_ATTEMPTS = 5;
    private String email;
    protected String token;
    private String secret;

    public User(String email) {
        this.email = email;
        this.token = "";
        this.secret = "shh";
    }

    public static String normalizeEmail(String email) {
        return email.toLowerCase();
    }

    public boolean login(String password) {
        return true;
    }

    public boolean equals(Object other) {
        return false;
    }
}
